src/txn/txn_rollback_to_stable.c

```diff
@@ -8,9 +8,8 @@
 
 #include "wt_internal.h"
 
-#define WT_CHECK_RECOVERY_FLAG_TS_TXNID(session, txnid, durablets)             \
-    ((durablets) == WT_TS_NONE && F_ISSET(S2C(session), WT_CONN_RECOVERING) && \
-      (txnid) >= S2C(session)->recovery_ckpt_snap_min)
+#define WT_CHECK_RECOVERY_FLAG_TS_TXNID(session, txnid) \
```

Comment:
Can we rename the macro to `WT_CHECK_RECOVERY_FLAG_TXNID` since we are not comparing the timestamp anymore. ?

Reply:
Fixed.